### PR TITLE
Feature/ccpp

### PR DESCRIPTION
**Pull request by lgannoaa**

### CODE REVIEW

File: parm/config/config.base.emc.dyn [hunk 1]
```diff
@@ -33,7 +33,6 @@ export SCRgfs=$HOMEgfs/scripts
 
 # GLOBAL static environment parameters
 export NWPROD="@NWPROD@"
-export COMROOT="@COMROOT@"
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.emc.dyn]: This should not be deleted.

File: parm/config/config.base.emc.dyn [hunk 2]
```diff
@@ -92,10 +94,10 @@ export BASE_JOB="$HOMEgfs/jobs/rocoto"
 
 # EXPERIMENT specific environment parameters
 export SDATE=@SDATE@
-export FDATE=@FDATE@
 export EDATE=@EDATE@
 export EXP_WARM_START="@EXP_WARM_START@"
 export assim_freq=6
+export FDATE=$($NDATE +$assim_freq $SDATE)
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.emc.dyn]: This should not be added...this is from an old commit. This is not in develop.

File: parm/config/config.base.emc.dyn [hunk 3]
```diff
@@ -92,10 +94,10 @@ export BASE_JOB="$HOMEgfs/jobs/rocoto"
 
 # EXPERIMENT specific environment parameters
 export SDATE=@SDATE@
-export FDATE=@FDATE@
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.emc.dyn]: This should not be getting deleted.

File: parm/config/config.base.emc.dyn [hunk 4]
```diff
@@ -202,10 +208,10 @@ export IAUFHRS="3,6,9"
 export IAU_FHROT=`echo $IAUFHRS | cut -c1`
 export IAU_DELTHRS=6
 export IAU_OFFSET=6
-export DOIAU_ENKF=${DOIAU:-"YES"}   # Enable 4DIAU for EnKF ensemble
+export DOIAU_ENKF="YES"   # Enable 4DIAU for EnKF ensemble
 export IAUFHRS_ENKF="3,6,9"
 export IAU_DELTHRS_ENKF=6
-if [[ "$SDATE" = "$CDATE" && $EXP_WARM_START = ".false." ]] || [[ "$DOIAU" = "NO" ]] ; then # Cold starting or IAU off
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.emc.dyn]: This line should not be changing. Again, please sync with develop.

File: parm/config/config.base.emc.dyn [hunk 5]
```diff
@@ -270,8 +280,7 @@ export netcdf_diag=".true."
 export binary_diag=".false."
 
 # Verification options
-export DO_METP="YES"         # Run METPLUS jobs - set METPLUS settings in config.metp
-export DO_VSDB="YES"         # Run VSDB package - set VSDB settings in config.vrfy
+export DO_METP="YES"                  # Run MET+ jobs
```
Comment (KateFriedman-NOAA) [inline parm/config/config.base.emc.dyn]: This section should not be changing.

File: parm/config/config.gldas [hunk 6]
```diff
@@ -12,5 +12,4 @@ export GLDASSH=$HOMEgfs/scripts/exgdas_atmos_gldas.sh
 export gldas_spinup_hours=72
 export CPCGAUGE=$DMPDIR
 
-
```
Comment (KateFriedman-NOAA) [inline parm/config/config.gldas:15]: Remove this extra line.

File: parm/config/config.resources [hunk 7]
```diff
@@ -115,13 +115,10 @@ elif [ $step = "anal" ]; then
...
     export npe_anal_gfs=1000
     if [ $CASE = "C384" ]; then
-      export npe_anal=400
-      export npe_anal_gfs=400
-    fi
-    if [ $CASE = "C192" -o $CASE = "C96" -o $CASE = "C48" ]; then
-      export npe_anal=84
-      export npe_anal_gfs=84
+      export npe_anal=160
+      export nth_anal=10
```
Comment (KateFriedman-NOAA) [inline parm/config/config.resources]: None of these changes should be happening...please sync with develop!

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]
Again...there are issues that were not resolved. Please sync your feature/ccpp branch with develop...there are recent changes committed to develop that are getting wiped out in this PR incorrectly. There are also questions I left in PR #184 that didn't get an answer.

PLEASE DO NOT CLOSE THIS PR! Please make fixes in your feature/ccpp branch in your fork, they will automatically show up in this PR when committed to your branch. Closing the PR and restarting it is unnecessary with GitHub. We want the log of questions/comments/changes.

**Comment by lgannoaa** [comment]
I tested this package on DELL with C768 as requested by Fanglin by using imp_physics=11.
Fanglin also requested to merge it this package to the develop without testing the imp_physics=8.
Arun told us this merge does not need to reproduce GFSv16 (current develop).

Can you help to test this on other platform as needed? 
User does not need a switch to build the unified forecast executable. It is build with FV3_GSD_v0,FV3_GSD_noah,FV3_GFS_v16beta.

The additional module is not needed. 
I modified this package by:
rm /gpfs/dell2/emc/modeling/noscrub/Lin.Gan/ecflow-dell/feature_ccpp/modulefiles/module_base.wcoss_dell_p3_fcst
rm /gpfs/dell2/emc/modeling/noscrub/Lin.Gan/ecflow-dell/feature_ccpp/ush/load_fv3gfs_modules_fcst.sh
Modify jobs/rocoto/fcst.sh to NOT use the additional module.

The file config.aeroic is not needed for this commit:
rm ush/rocoto/setup_workflow_fcstonly_aeroic.py parm/config/config.aeroic jobs/rocoto/aeroic.sh

config.base.emc.dyn changes need because there are three suites in this configuration. The FV3_GFS_v16beta is one of them. Re-sync with develop.

Comment line in config.resources removed. Re-sync with develop.

build_fv3.sh cleanup is done.

Comment line in checkout.sh removed.

Remove extra line in link_fv3gfs.sh.

**Comment by WalterKolczynski-NOAA** [comment]
@KateFriedman-NOAA requested this PR not be closed, so I am going to reopen it pending a note of approval from her on this PR that it can be closed without being merged.

**Review by KateFriedman-NOAA — APPROVED** [review]
Initial round of CCPP changes look good. Tests were run on Orion, Hera, and WCOSS-Dell. Caveat: can't run CCPP with waves on yet, further commit will remedy this.

**Comment by KateFriedman-NOAA** [comment]
This PR will close issue #187 .

**Pull request merged**